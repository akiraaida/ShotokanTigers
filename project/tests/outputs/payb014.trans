10 Ashe Ace             00000 00000.00 S 
03 Ashe Ace             12348 00100.00 TV
01 Ashe Ace             12348 00000.10   
03 Ashe Ace             12348 01900.00 TV
01 Ashe Ace             12348 00000.10   
00                      00000 00000.00   
