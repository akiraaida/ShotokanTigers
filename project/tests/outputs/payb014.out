Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
ERROR, THE VALUE ENTERED IS BEYOND THE PAYBILL LIMIT.
Available Balance (12348):
10010.00
Daily Payment Limit to TV Remaining (12348):
2000.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12348):
9909.90
Daily Payment Limit to TV Remaining (12348):
1900.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
ERROR, THE VALUE ENTERED IS BEYOND THE PAYBILL LIMIT.
Available Balance (12348):
9909.90
Daily Payment Limit to TV Remaining (12348):
1900.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
Amount has been paid successfully.
Available Balance (12348):
8009.80
Daily Payment Limit to TV Remaining (12348):
0.00
Please enter a command: 
Please enter the user's account number: 
Please enter recipient to be paid: 
Please enter amount to pay: 
ERROR, THE VALUE ENTERED IS BEYOND THE PAYBILL LIMIT.
Available Balance (12348):
8009.80
Daily Payment Limit to TV Remaining (12348):
0.00
Please enter a command: 
